extensions/protocols/dcp/dcp-issuer/dcp-issuer-spi/src/main/java/org/eclipse/edc/identityhub/protocols/dcp/issuer/spi/model/CredentialObject.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *  Copyright (c) 2025 Cofinity-X
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Cofinity-X - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.identityhub.protocols.dcp.issuer.spi.model;
+
+import org.eclipse.edc.iam.verifiablecredentials.spi.model.presentationdefinition.PresentationDefinition;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CredentialObject {
+
+    public static final String CREDENTIAL_OBJECT_TERM = "CredentialObject";
```

Comment:
don't we usually prefix the properties with the namespace?

Reply:
We usually do but for protocol messages since we might support different protocol version/ namespaces we apply the `ns` in the transformers depending on the protocol version as we do for `DSP`